packages/caip/src/assetId/assetId.ts

```diff
@@ -160,5 +160,12 @@ export const fromAssetId: FromAssetId = assetId => {
   }
 }
 
+export const deserializeNftAssetReference = (
+  assetReference: string,
+): [address: string, id: string] => {
+  const [address, id] = assetReference.split('/')
```

Comment:
Not sure if `address` is a good terminology within a caip context, but felt weird to destructure an asset reference out of an asset reference

Reply:
ha thought the same thing, but I think this makes the most sense in this case